docs/contents/tutorials/01_getting_started.rst

```diff
@@ -103,7 +113,7 @@ So let's create a ``clicks-receiver`` subscription:
 
 .. code-block:: bash
 
-    curl -d '{"name": "clicks-receiver", "endpoint": "http://requestb.in/1isy54g1", "supportTeam": "my-team"}' -H "Content-Type: application/json" http://192.168.59.103:8080/topics/com.example.events.clicks/subscriptions
+    curl -d '{"name": "clicks-receiver", "endpoint": "http://requestb.in/1isy54g1", "supportTeam": "my-team"}' -H "Content-Type: application/json" http://:8090/topics/com.example.events.clicks/subscriptions
```

Comment:
fixed the curl line, thanks.

...and yeah, we're still at v0.8.0 with docker images (http://box.allegrotech.io/hermes/), `supportTeam` is since v0.8.1.